cli/crates/cli/tests/dev_watch.rs

```diff
@@ -81,3 +80,29 @@ async fn dev_watch() {
         assert!(generated_types_path.is_file());
     }
 }
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn dev_watch_with_custom_codegen_path() {
+    let mut env = Environment::init();
+
+    env.grafbase_init(GraphType::Single);
```

Comment:
Shouldn't this be `Standalone`? Weird that we didn't change it

Reply:
Hmm true, let me check that out in another branch.

Reply:
PR coming